### ISSUE TITLE
[BUG] Executing a query against a table with a varbinary(max) datatype always returns an empty results.

### ISSUE THREAD

**Issue by lroal** [markdirish/node-odbc#269]
Describe your system
ODBC Driver: ODBC 17 Microsoft SQL Windows Driver
Database Name: rdbDemo
Database Version: Microsoft SQL Server 2019 (15.0.4223.1) 64bit 
Database OS: Microsoft windows 10 pro
Node.js Version: 16.14.2
Node.js OS: Windows 10 pro

Describe the bug
When doing a select or executing a procedure that calls a SELECT on a table with a datatype of varbinary(max) the results is always empty.

Expected behavior
Get a results regardless of data types defined in table columns.
I have table called _customer with two colums: id and picture. The picture column is a varbinary(max). If I omit the picture column, everything is ok.  It also returns no result if using a nvarchar(max) column.

result:
```
[
  statement: "select TOP 1 _customer.id as s_customer0,_customer.picture as s_customer1 from _customer _customer where _customer.id='a0000000-0000-0000-0000-000000000000' order by _customer.id", 
  parameters: [],
  return: undefined,
  count: 1,
  columns: [
    {
      name: 's_customer0',
      dataType: -11,
      columnSize: 36,
      decimalDigits: 0,
      nullable: false
    },
    {
      name: 's_customer1',
      dataType: -3,
      columnSize: 0,
      decimalDigits: 0,
      nullable: true
    }
  ]
]
```

**Comment by kadler** [comment]
Looks like it returns 0 for columnSize form SQLDescribeCol, which probably confuses something somewhere (I would have thought it would have given back rows without that column, or just empty values. No rows at all is rather strange).

In fact, it confused me but looking at the docs for SQLDescribeCol, it is actually documented:

> If the column size cannot be determined, the driver returns 0.

https://docs.microsoft.com/en-us/sql/odbc/reference/syntax/sqldescribecol-function

Presumably this only occurs for "long" data, so we could go down the long_data route using SQLGetData, but that's not always possible (eg. if we're doing block fetch) so we need to determine a fallback binding length. One way would be to call SQLGetTypeInfo with the data type we were given and use the COLUMN_SIZE returned (but it wouldn't surprise me if some drivers don't support this either). Another solution would be to pick a reasonable value and let the user override it (maybe just use initialBufferSize?)

**Comment by lroal** [comment]
Thank's for looking at the problem. Maybe you can pick up some ideas here, https://github.com/TimelordUK/node-sqlserver-v8 ?
I guess he faces the same problem and has solved it somehow.

**Comment by markdirish** [comment]
Stalebot should calm down. There is a PR ready for review for this issue: #271

**Comment by lroal** [comment]
Do you know when this is going to be released ?

**Comment by markdirish** [comment]
Was fixed in #271 and is live in v2.4.5
